app/models/area.rb

```diff
@@ -3,6 +3,7 @@ class Area
   field :name, type: String
   field :pincode, type: Integer
   field :city_id, type: String
+  field :user_id, type: String
```

Comment:
why explicit ids and why not association like belongs_to :user or belongs_to :owner,class_name: User ...